src/operations/campaigns/sponsored-products-campaign-operation.ts

```diff
@@ -57,7 +57,7 @@ export class SponsoredProductsCampaignOperation extends Operation {
    * @returns
    */
   @Decode(CampaignExtended)
-  public getCampaignEx(campaignId: CampaignId) {
+  public getCampaignExtended(campaignId: CampaignId) {
```

Comment:
These changes go beyond docs.

In fact, this is a breaking change.

I am not opposed to the change, but need to mark it as so. See conventional commit docs.

Reply:
Thanks for your comments! I added a `BREAKING CHANGE:` in commit message body.